tests/acceptance/test_cases/acceptance_test_case.py

```diff
@@ -0,0 +1,55 @@
+"""
+Base class for acceptance tests of the API. Acceptance tests run on the
+entire server stack, and check that TopChef works end-to-end
+"""
+import unittest
+from topchef import APP_FACTORY
+from topchef.database.schemas.database_schema import DatabaseSchema
+import multiprocessing as mp
+
+
+class AcceptanceTestCase(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.host = 'localhost'
+        cls.port = 60123
+        cls.protocol = 'http'
+        cls.app = APP_FACTORY.app
+        cls.app_process = mp.Process(target=cls.app.run)
+        cls.app_process.daemon = True
+        cls.app_process.start()
+
+        cls._create_database(APP_FACTORY.engine)
+
+    @classmethod
+    def tearDownClass(cls):
+        cls._drop_database(APP_FACTORY.engine)
+
+        if hasattr(cls, 'app_process'):
+            cls.app_process.terminate()
+            _ = cls.app_process.exit_code
```

Comment:
Use [``cls.app_process.join()``](https://docs.python.org/2/library/multiprocessing.html#multiprocessing.Process.join) instead. See the [example for multiprocessing](https://docs.python.org/2/library/multiprocessing.html#the-process-class).